crates/gen-rust-lib/Cargo.toml

```diff
@@ -11,3 +11,7 @@ test = false
 [dependencies]
 wit-bindgen-core = { workspace = true }
 heck = { workspace = true }
+
+[features]
+default = ["std"]
+std = []
```

Comment:
Reading over this again, I think a compile-time option may not be the best way to shoehorn this feature for the code generators. Could the `print_typedef_enum` method grow a `gen_error: bool` method perhaps which is threaded through? For the Wasmtime-host generator this would always pass `true` but for the Rust-guest generator I think it would be best to have this as a configurable `Opts` option which, in the proc-macro, would be configured based on a feature flag of the proc-macro itself.

Reply:
I think this would help cut down on the `default-features = false` traffic as well.

Reply:
Done; and yeah, it's a lot simpler that way.